driver/modern_bpf/helpers/store/auxmap_store_params.h

```diff
@@ -1385,3 +1384,171 @@ static __always_inline void auxmap__store_fdlist_param(struct auxiliary_map *aux
 	/* The param size is: 16 bit for the number of pairs + size of the pairs */
 	push__param_len(auxmap->data, &auxmap->lengths_pos, sizeof(u16) + (num_pairs * (sizeof(s64) + sizeof(s16))));
 }
+
+static __always_inline void apply_dynamic_snaplen(struct pt_regs *regs, u16 *snaplen, bool only_port_range)
+{
+	if(!maps__get_do_dynamic_snaplen())
+	{
+		return;
+	}
+
+	/* Please note that we can use this helper also for syscalls not related to the network.
```

Comment:
I hope this is clear, but the difference between the modern probe and the other 2 drivers is that these syscalls:
```
 - readv
 - preadv
 - writev 
 - pwritev
 - recvmsg
 - sendmsg
```
use only the `port_range` detection logic, see the `only_port_range` flag